bugwarrior/services/http.py

```diff
@@ -0,0 +1,75 @@
+import requests
+import logging
+from dateutil.parser import isoparse
+
+from bugwarrior.services import IssueService, Issue
+
+log = logging.getLogger(__name__)
+
+
+class HttpIssue(Issue):
+    UUID = 'http_uuid'
+    URL = 'http_url'
+
+    UNIQUE_KEY = (UUID,)
+    UDAS = {
+        URL: {
+            'type': 'string',
+            'label': "API URL"
+        },
+        UUID: {
+            'type': 'string',
+            'label': 'Virtual API UUID for task'
+        }
+    }
+
+    def get_default_description(self):
+        return self.record.get(
+            'description',
+            self.record.get(
+                'uuid',
+                self.extra.get('url')
+            )
+        )
+
+    def to_taskwarrior(self):
+        return {
+            'entry': isoparse(self.record.get('entry')) if self.record.get('entry') else None,
+            'tags': self.record.get('tags', []),
+            'project': self.get_project(),
+            'description': self.record.get('description'),
+            'priority': self.get_priority(),
+
+            self.UUID: self.record.get('uuid'),
+            self.URL: self.extra.get('url')
+        }
+
+
+class HttpService(IssueService):
+    APPLICATION_NAME = 'Bugwarrior HTTP Service'
```

Comment:
What is the APPLICATION_NAME attribute for?

Reply:
I don't know, will remove